manual/src/cmds/Makefile

```diff
@@ -28,7 +28,7 @@ all: $(FILES)
 	mv $*.texquote_error.tex $@
 
 warnings-help.etex: ../warnings-help.etex
-	cp $< $@
+	sed 's/@/\\@/g' $< > $@
```

Comment:
this looks a bit dubious to me; isn't the file used somewhere else? (otherwise it would be defined in the present directory, right?). I would `sed --inplace` on the file and commit that.

Reply:
The file is generated by `manual/src/Makefile` from `warnings.ml`. You are right that it is probably to amend the generation there. I was under the impression that the file was used more than once, but this doesn't seem to be the case.